src/server/journal/pending_buf.h

```diff
@@ -20,7 +20,7 @@ class PendingBuf {
 #ifdef UIO_MAXIOV
     static constexpr size_t kMaxBufSize = UIO_MAXIOV;
 #else
-    static constexpr size_t kMaxBufSize = 1024;
+    static constexpr size_t kMaxBufSize = 1500;  // 1024;
```

Comment:
remove `// 1024;`

Reply:
I will. I'm considering this to be flag converted to flag which can be set.

Reply:
Added flag to control threshold